posthog/api/test/test_insight.py

```diff
@@ -1366,6 +1366,32 @@ def test_another_user_viewing_an_insight_does_not_impact_the_list(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(response_data["results"]), 0)
 
+    def test_get_recent_insights_with_feature_flag(self):
+        filter_dict = {"events": [{"id": "$pageview"}], "breakdown": "$feature/insight-with-flag-used"}
+        filter_dict2 = {
+            "events": [{"id": "$pageview"}],
+            "properties": [{"key": "$active_feature_flag", "value": "insight-with-flag-used"}],
+        }
+        filter_dict3 = {"events": [{"id": "$pageview"}], "breakdown": "email"}
```

Comment:
@neilkakkar so then this should address the problem then right? Unless you were asking for me to split this up into another test

Reply:
Nope, exactly, for the same test itself.

Reply:
This looks good